pkg/tweetnacl/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=tweetnacl
+PKG_URL=https://github.com/kYc0o/tweetnacl
+PKG_VERSION=3548e2c0474731123e8e22a8049a945cd227aea6
+
+.PHONY: all
+
+all: git-download
+	@cp Makefile.tweetnacl $(PKG_BUILDDIR)/Makefile
```

Comment:
If you now opened your own Repo, why not add a commit to that that does that for you. Should be pretty easy to maintain (as long as tweetnacl doesn't add their own Makefile ;-))